packages/lisk-transactions/src/transactions/0_transfer.ts

```diff
@@ -298,7 +297,7 @@ export class TransferTransaction extends BaseTransaction {
 
 	protected applyAsset(store: StateStore): ReadonlyArray<TransactionError> {
 		const errors: TransactionError[] = [];
-		const sender = store.get<Account>(ENTITY_ACCOUNT, 'address', this.senderId);
+		const sender = store.account.get<Account>('address', this.senderId);
```

Comment:
probably we can remove the generics, since account is clear
```
store.account.get('address', this.senderId);
```

Reply:
typescript does not recognize return value as account without it :/